researchstudio-sat/webofneeds
# Refactored number/range picker and sparql-query generation, fixed some rdf-types

to test:

1. start creating needs using all numerical inputs (e.g. custom need, offer/search real estate). one should be a "search real estate"-need.
2. fold details in and out during the process and check the text-summaries
3. post the need 
4. look at RDF -- all properties using schema.org (`s:...`) should have values typed with e.g. `s:String`, except for the `xsd:float` where we have to break with the ontology as our matcher can only match `xsd:float` atm.
5. look at the SPARQL-query in the "search real estate"-need. all details should have contributed to the query and should appear there, all variables in the filter expressions should be bound in the graph pattern, all used prefixes should be declared.
6. look at the post-detail view of the generated needs, all details should show up there as well.

please leave the branch open after merging. this is just an early PR so other ppl working on the use-cases can use the new util-functions, and to lessen/avoid later merge-conflicts.
